dinesykmeldte-backend/src/main/kotlin/no/nav/syfo/minesykmeldte/MineSykmeldteService.kt

```diff
@@ -151,16 +161,52 @@ class MineSykmeldteService(
     }
 
     suspend fun markSykmeldingRead(sykmeldingId: String, lederFnr: String): Boolean {
-        return mineSykmeldteDb.markSykmeldingRead(sykmeldingId, lederFnr)
+        return mineSykmeldteDb.markSykmeldingRead(sykmeldingId, lederFnr).also {
+            updateReadCountKafkaTopic(lederFnr)
+        }
     }
 
     suspend fun markSoknadRead(soknadId: String, lederFnr: String): Boolean {
-        return mineSykmeldteDb.markSoknadRead(soknadId, lederFnr)
+        return mineSykmeldteDb.markSoknadRead(soknadId, lederFnr).also {
+            updateReadCountKafkaTopic(lederFnr)
+        }
     }
 
     suspend fun markHendelseRead(hendelseId: UUID, lederFnr: String): Boolean {
-        return mineSykmeldteDb.markHendelseRead(hendelseId, lederFnr)
+        return mineSykmeldteDb.markHendelseRead(hendelseId, lederFnr).also {
+            updateReadCountKafkaTopic(lederFnr)
+        }
+    }
+
+    suspend fun updateReadCountKafkaTopic(lederFnr: String) {
+        val mineSykmeldte = getMineSykmeldte(lederFnr)
+
+        return mineSykmeldte.forEach { sykmeldt ->
+            nlReadCountProducer.send(
+                NLReadCountKafkaMessage(
+                    KafkaMetadata(
+                        timestamp = OffsetDateTime.now(ZoneOffset.UTC),
+                        source = "leder"
+                    ),
+                    NLReadCount(
+                        narmestelederId = sykmeldt.narmestelederId,
+                        unreadSykmeldinger = sykmeldt.sykmeldinger.filter { !it.lest }.size,
+                        unreadSoknader = sykmeldt.previewSoknader.filter { isSoknadUnread(it) }.size,
+                        unreadMeldinger = sykmeldt.aktivitetsvarsler.filter { it.lest == null }.size,
+                        unreadDialogmoter = sykmeldt.dialogmoter.size,
+                        unreadOppfolgingsplaner = sykmeldt.oppfolgingsplaner.size,
+                    )
+                )
+            )
+        }
     }
+
+    private fun isSoknadUnread(soknad: PreviewSoknad): Boolean =
+        when (soknad) {
+            is PreviewSendtSoknad -> !soknad.lest
+            is PreviewNySoknad -> if (soknad.ikkeSendtSoknadVarsel) !soknad.lest else false
```

Comment:
Kan lederen lese en søknad som er ny og ikke sendt? Trodde den måtte være sendt for at den skal kunne dukke opp som noe lederen kan se på?